backend/actions/address_detail.js

```diff
@@ -133,4 +133,6 @@ app.get('/address/:address', (req, res) => {
     console.log(`Error retrieving information for addresss - ${urlAddress}. Error Message - ${err.message}`);
     logger.error(`Error retrieving information for addresss - ${urlAddress}. Error Message - ${err.message}`);
   });
-})
\ No newline at end of file
+})
+
+module.exports = {cl, elect};
```

Comment:
Why export these?

Reply:
They are required for stubbing the instance while testing, Sir